src/QMCWaveFunctions/Fermion/DiracDeterminantBase.h

```diff
@@ -133,10 +133,10 @@ class DiracDeterminantBase : public WaveFunctionComponent
   }
 
   // Stop makeClone
-  WaveFunctionComponentPtr makeClone(ParticleSet& tqp) const override final
+  std::unique_ptr<WaveFunctionComponent> makeClone(ParticleSet& tqp) const override
```

Comment:
This was intentionally made final.

Reply:
Probably to add this comment "final keyword is intended to disable makeClone being further inherited"

Reply:
OK, I'll change it back to final.